conjur/controller/hostfactory_controller.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+from conjur.errors import MissingRequiredParameterException
+
+"""
+VariableController module
+
+This module is the controller that facilitates all list actions
+required to successfully execute the HostFactory commands
+"""
+import sys
+
+from conjur.data_object.create_token_data import CreateTokenData
+from conjur.logic.hostfactory_logic import HostFactoryLogic
+
+
+# pylint: disable=too-few-public-methods
+class HostFactoryController:
+    """
+    HostFactoryController
+
+    This class represents the Presentation Layer for the HostFactory command
+    """
+
+    def __init__(self, hostfactory_logic: HostFactoryLogic):
+        self.hostfactory_logic = hostfactory_logic
+
+    def create_token(self, create_token_data: CreateTokenData):
+        """
+        Method that facilitates create token call to the logic
+        """
+        if create_token_data is None:
+            raise MissingRequiredParameterException('create_token_data')
```

Comment:
For consistency maybe something like `Token data is missing`? this is for the UI. Even though I have to say that your way is much easier to debug. @sigalsax maybe we should use this approach for all our `MissingRequiredParameterException ` ?

Reply:
@mbenita-Cyberark agreed. This error isn't enough to explain what happened. It will just print out 'create_token_data'. Please a more explanatory error of what happened at tag Shuli

Reply:
@sigalsax his should not happen to user unless a developer did something wrong.